sys/net/gnrc/routing/rpl/srh/gnrc_rpl_sr_table.c

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (C) 2025 Ahmad Mahmod <ahmad.nawras.mahmod@gmail.com -- mahmod@unistra.fr> *
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include "net/gnrc/rpl/sr_table.h"
+#include "net/gnrc/rpl/dodag.h"
+#include "net/ipv6.h"
+#include "net/ipv6/addr.h"
+
+#define ENABLE_DEBUG  0
+#include "debug.h"
+
+static fib_table_t table;
+static fib_entry_t _sr_table_entries[GNRC_SR_FIB_TABLE_SIZE];
+
+static char addr_str[IPV6_ADDR_MAX_STR_LEN];
+static ipv6_addr_t root_node = { { 0 } };
+
+void gnrc_sr_initialize_table(ipv6_addr_t *root, kernel_pid_t iface)
+{
+    (void)iface;
+    if (root != NULL) {
+        memcpy(&root_node, root, sizeof(ipv6_addr_t));
+    }
+    else {
+        DEBUG("RPL SR Table: FIB table not initialized: invalid root address\n");
+    }
+
+    table.size = GNRC_SR_FIB_TABLE_SIZE;
+    table.table_type = FIB_TABLE_TYPE_SH;
+    table.data.entries = _sr_table_entries;
```

Comment:
Shouldn't it be `FIB_TABLE_TYPE_SR` and `table.data.source_routes` since it's a table for source routing and not single hop entries?
What am I missing?